File: app/dbmodels/AssetDisplay.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'AssetDisplay',
    'menu'          => ''
);
$table['fields'] = array(
    'id'               => DBModels::type('increments', 
                                         array('visible' => false)),
    'asset_id'        => DBModels::type('integer', 
                                         array('visible' => false)),
    'size'             => DBModels::type('integer'),
    'display_type_id'  => DBModels::type('integer'),
    'display_model_id' => DBModels::type('integer'),
    'have_subd'        => DBModels::type('boolean'),
    'have_bnc'         => DBModels::type('boolean'),
    'have_vga'         => DBModels::type('boolean'),
    'have_dvi'         => DBModels::type('boolean'),
    'have_displayport' => DBModels::type('boolean'),
);

$table['relationships'] = array(
    'asset'     => array(
        'type'  => 'belongsTo',
        'item'  => 'Asset'),
    'display_type'  => array(
        'type'  => 'belongsTo',
        'item'  => 'DisplayType'),
    'display_model'  => array(
        'type'  => 'belongsTo',
        'item'  => 'DisplayModel')
);

File: fusionglpi/database/asset_display.py
from fusionglpi import db

class AssetDisplay(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    asset_id = db.Column(db.Integer, db.ForeignKey('asset.id'))
    size = db.Column(db.Integer, nullable=False, server_default="0")
    display_type_id = db.Column(db.Integer, db.ForeignKey('display_type.id'))
    display_model_id = db.Column(db.Integer, db.ForeignKey('display_model.id'))
    have_subd = db.Column(db.Boolean, nullable=False, server_default="0")
    have_bnc = db.Column(db.Boolean, nullable=False, server_default="0")
    have_vga = db.Column(db.Boolean, nullable=False, server_default="0")
    have_dvi = db.Column(db.Boolean, nullable=False, server_default="0")
    have_displayport = db.Column(db.Boolean, nullable=False, server_default="0")
